db/schema.rb

```diff
@@ -15,7 +15,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
-  enable_extension "pg_stat_statements"
 
```

Comment:
@schneems Done in https://github.com/codetriage/codetriage/commit/87cb81047a9268abffb7030cb849abbb2dc25bed